### ISSUE TITLE
node_client_example/src/client.js has issues with authorization.

### ISSUE THREAD

**Issue by dockter-84** [jupyter/kernel_gateway_demos#27]
node_client_example/src/client.js has issues with authorization.

Currently the username and password are hardcoded in the example: https://github.com/jupyter/kernel_gateway_demos/blob/b4d7427639dce2f3e22940d0156738786db98b51/node_client_example/src/client.js#L34

If the values of `BASE_GATEWAY_HTTP_URL` and `BASE_GATEWAY_WS_URL`  are constructed with `https://username:password@host/`, the username and password in the URL are ignored. 

A suggestion from Peter Parente:  "...make the node client accept the username password from environment variables instead of hardcoding them. Then it can be controlled externally without modifying source in the open source..."

**Comment by parente** [comment]
@dockter-84 Happy to take a PR if you have it working.

**Comment by dockter-84** [comment]
example.py
```python
# Copyright (c) Jupyter Development Team.
# Distributed under the terms of the Modified BSD License.
import pyspark

if not globals().has_key('sc'):
    sc = pyspark.SparkContext()
rdd = sc.parallelize(range(1000))
sample = rdd.takeSample(False, 5)
print(sample)
```

**Comment by parente** [comment]
Fixed by #32.
